RPIPs/RPIP-43.md

```diff
@@ -8,37 +8,95 @@ status: Draft
+  - Dissolved: this validator's withdrawal credentials have been discovered to
+               be incorrect (ie, not this megapool)
+
+Node operators can manage the set of validators in their megapool:
+
+- A Node Operator SHALL be able to add new validators to their megapool
+- A Node Operator SHALL be able to migrate existing validators to their megapool
+- New and existing validators' withdrawal credentials MUST be checked to
+  correctly refer to the megapool address, with the validator status set to
+  Deposited or Dissolved accordingly
+     - The check MAY occur after the validator is added initially as Deposited
+- A Node Operator SHALL be able to remove exited validators from their megapool
+
+### Funds Management
+
+This subsection needs some work. I think it should say who can trigger
+distribution of funds out of the megapool, whether those funds are pending
+deposits, rewards, or remaining principal. This is a non-trivial question since
+funds do not necessarily belong to the node operator. There could be a link out
+to another RPIP with the details.
+
 - A Node Operator SHALL be able to distribute rewards from all validators in their megapool at once
   - This MAY be temporarily blocked while validators are exiting or pending removal
-- Legacy minipool deposits (ie, deposits not in megapools) SHALL be disabled
-- RPL MAY be staked to a megapool
-  - Validators MAY be added to a megapool regardless of RPL staked to the megapool (note that this means ETH-only node operation is possible)
-- Legacy RPL staking (ie staking RPL to a node rather than to a megapool) SHALL be disabled
-- A user MAY set any amount of RPL "Staked" to a megapool to "Unstaking"
-- Once RPL has been "Unstaking" for `unstaking_period` days, it can be withdrawn (this setting is defined in [RPIP-30](RPIP-30.md))
-- All RPL that was "Unstaking" MUST be withdrawn at one time
-- There SHALL only be a single time stored for "Unstaking" RPL; this means that if a user sets
-  additional RPL to "Unstaking" after some RPL is already "Unstaking", they must wait for
-  `unstaking_period` before any is available to withdraw
-  - All RPL that was already "Unstaking" for `unstaking_period` SHOULD be withdrawn when a user
-    sets additional RPL to "Unstaking"
+
+### RPL Staking
+
+This RPIP includes a rework of the treatment of RPL staked on Rocket Pool
+nodes. We refer to RPL staked for a node prior to the implementation of this
+RPIP as "legacy staked RPL", and to RPL staked according to the rules described
+in this RPIP as, simply, "staked RPL", or as "megapool staked RPL" if otherwise
+ambiguous.
+
+Megapool staked RPL (in the protocol, ie, before being withdrawn) can be in one
+of two states:
+- Staked: the RPL has been deposited and staked (locked) in the protocol, for a
+          node with a megapool deployed
+- Unstaking: the RPL has been marked as intended to be withdrawn
+
+Rules specifying the movement of staked RPL are as follows:
```

Comment:
This is a good example. Per the earlier definition, this means "megapool staked RPL". However, this is about the superset of megapool staked RPL and legacy staked RPL.